src/main/java/edu/wisc/my/messages/controller/MessagesController.java

```diff
@@ -0,0 +1,54 @@
+package edu.wisc.my.messages.controller;
+
+import org.springframework.web.bind.annotation.RestController;
+import org.json.JSONObject;
+import edu.wisc.my.messages.service.MessagesService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@RestController
+public class MessagesController {
+    protected final Logger logger = LoggerFactory.getLogger(getClass());
+    private MessagesService messagesService;
+    
+    @RequestMapping(value="/json")
```

Comment:
I would favor maybe /messages?  /json doesn't really tell me what I'm requesting.  Maybe even adding in `method = RequestMethod.GET` as well.